osmaxx-py/osmaxx/excerptexport/services/conversion_api_client.py

```diff
@@ -168,18 +176,11 @@ def update_order_status(self, extraction_order):
         job_status = self.job_status(extraction_order)
 
         if job_status:
-            if job_status['status'] == 'done' and job_status['progress'] == 'successful':
-                if not extraction_order.state == ExtractionOrderState.FINISHED and \
-                   not extraction_order.state == ExtractionOrderState.FAILED:
-                    self.download_result_files(extraction_order)
-                    extraction_order.state = ExtractionOrderState.FINISHED
-                    extraction_order.save()
-            elif job_status['status'] == 'started':
-                extraction_order.state = ExtractionOrderState.PROCESSING
-                extraction_order.save()
-            return True
-        else:
-            return False
+            progress = job_status['progress']
+            extraction_order.set_status_from_conversion_progress(progress)
+            extraction_order.save()
+            if progress == 'successful':
+                self._download_result_files(extraction_order, job_status)
```

Comment:
This change turns the docstring wrong. (Function doesn't return anything anymore.)